core/src/main/java/edu/wpi/grip/core/operations/networktables/NTPublishOperation.java

```diff
@@ -49,16 +50,20 @@ public NTPublishOperation(Class<S> socketType, Class<T> reportType, Function<S,
         this.type = checkNotNull(socketType, "Type was null");
         this.converter = checkNotNull(converter, "Converter was null");
 
-        // Any accessor method with an @NTValue annotation can be published to NetworkTables.
-        for (Method method : reportType.getDeclaredMethods()) {
-            if (method.getAnnotation(NTValue.class) != null) {
-                if (method.getParameters().length > 0) {
+        // Any accessor method with an @NTValue annotation can be published to NetworkTables.  We sort them by their
+        // "weights" in order to avoid the issue of different JVM versions returning methods in a different order.
+        this.ntValueMethods = ImmutableList.copyOf(Arrays.asList(reportType.getDeclaredMethods()).stream()
+                .filter(method -> method.getAnnotation(NTValue.class) != null)
+                .sorted(Comparator.comparing(method -> method.getAnnotation(NTValue.class).weight()))
+                .iterator());
```

Comment:
If there are two methods with the same weight throw an `IllegalArgumentException`